app/views/layouts/frontend.html.erb

```diff
@@ -8,6 +8,7 @@
     <!-- Internet Explorer use the highest version available -->
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <%= yield(:meta_tags) if content_for? :meta_tags %>
+    <%= render partial: 'resources/home_meta_tags' if current_page?('/') -%>
```

Comment:
@nicktiberi On line 10, there's a yield, which is basically saying, if you have a content block that you've called `meta_tags`, I'm going to let it through right here. You can read up on on it if you like https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method

So, what you could do here is edit `app/views/pages/home.html.erb` and add something like:

``` ruby
<% content_for :meta_tags do %>
  <meta name="description" content="<%= t('home.intro.content') %>">
<% end %>
```

And that ought to do the trick. If this doesn't make sense, let me know and we can pair up on it.

Reply:
Learning something new about Rails every day. 🤓 Thanks!